### ISSUE TITLE
Exclusive Fullscreen mode broken on the latest Yuzu 1867

### ISSUE THREAD

**Issue by pkNurseJoy** [yuzu-emu/yuzu#6589]
Hello. I've been using Yuzu for a long time, and one thing I liked about it was the fact that Exclusive Fullscreen was functioning and working well. It is important to me as I have a very high refresh rate monitor "240hz" and when running 60FPS games the monitor will flicker at 240hz. So with Vsync off at random moments a duplicate frame is rendered causing a tiny stutter. Gsnyc or Freesync fixes that by having the GPU adjust the monitors refresh rate to match its frame output. Issue is, it only works in exclusive fullscreen which for some reason isn't working on the latest update. 

As you can see below the first picture shows the refresh rate my monitor is running at. If exclusive fullscreen was working that will allow Gsync to work and it would show "60" instead. 
![as](https://user-images.githubusercontent.com/66507357/125152619-aabfdf80-e11b-11eb-8af2-885c39ac9d72.jpg)

Similar here, the keyboard UI does not even work unless its borderless fullscreen. Another reason I know why Exclusive fullscreen is broken on the latest Yuzu. Anyway you guys can fix it?
![image0](https://user-images.githubusercontent.com/66507357/125152622-ad223980-e11b-11eb-9f0e-766c22baf56d.jpg)

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/6794695/yuzu_log.txt.old.txt)
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/6794696/yuzu_log.txt)

**Comment by AndroosYZ** [comment]
i've also had that issue as well. same type of display with the same refresh rate too. would love it if you guys fixed it. :)

**Comment by Future2K21** [comment]
I can also concur with this issue. The number of my framerate hasn't budged when both out and in fullscreen using Exclusive Fullscreen. 

![](https://cdn.discordapp.com/attachments/793194504462401547/863487925585051648/20210710_143227.jpg)

![](https://cdn.discordapp.com/attachments/793194504462401547/863488152211685414/20210710_143336.jpg)

![](https://cdn.discordapp.com/attachments/793194504462401547/863488294251397120/20210710_143410.jpg)

**Comment by Future2K21** [comment]
In addition to my previous post, I have discovered that the Adaptive/G-Sync features only activate on OpenGL's GLSL shader backend while not activating on Vulkan's. The pics from before are Vulkan's, here are GLSL's. 

![20210710_190422](https://user-images.githubusercontent.com/87243562/125178437-d17d2500-e1b2-11eb-80e0-0e884230978b.jpeg)

![20210710_190431](https://user-images.githubusercontent.com/87243562/125178446-ebb70300-e1b2-11eb-9e36-9655d3c231e7.jpeg)

**Comment by phly95** [comment]
Indeed. Playing Super Mario Odyssey with an Nvidia RTX 3060 Ti in Vulkan on Windows results in skipped frames, frame pacing issues and judders seemingly caused by not having exclusive fullscreen work properly. On Linux all those issues are non existent (if you force vsync/full composite in the Nvidia settings).
